test/models/customer_test.rb

```diff
@@ -0,0 +1,34 @@
+require "test_helper"
+
+describe Customer do
+  let(:customer_one) { customers(:customer_one)}
+  let(:customer) { Customer.new }
+
+  it "must be valid" do
+    customer_one.save
+    customer_one.valid?.must_equal true
+  end
+
+  it "must have a name, registered_at, address, city, state, postal code, and phone number" do
+    customer.valid?.must_equal false
+
+    customer.postal_code = "98119"
+    customer.valid?.must_equal false
+
+    customer.name = "Namo"
+    customer.valid?.must_equal false
+
+    customer.phone = "416173"
+    customer.valid?.must_equal false
+
+    customer.registered_at = "Tuesday, 03 Nov 2016 15:39:10 -0900"
+    customer.address = "whatever st"
+    customer.city = "Radtown"
+    customer.state = "WAtever"
+
+    customer.valid?.must_equal true
```

Comment:
I know it's a pain, but it's way more accurate to have separate tests that test each field on their own!

Start with a valid `customer` (like `customer_one` in this test), and then in each test take away one field and test that `customer_one.valid?.must_equal false`. Here, you started with an invalid object and kept adding until it was valid-- try going in the opposite direction.